### PR TITLE
Update to 1.1.4, use -O1

### PR DESCRIPTION
**Pull request by xhochy**
Unit tests fail with -O2 but work with -O1. This is still about twice as fast as the current binary artefact, -O2 seems to add another 2x but at least on macOS there is a problem as the unit tests fail reproducibly.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -9,8 +9,9 @@
 #
 source activate "${CONDA_DEFAULT_ENV}"
 
+autoreconf --force --install
 ./autogen.sh
-./configure --with-pic --prefix=$PREFIX
+CXXFLAGS="${CXXFLAGS} -O1 -DNDEBUG" ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
```
Comment (wesm) [inline recipe/build.sh]: Do you want to do `-O2` on Linux? You can do in bash

`if [ "$(uname)" == "Linux" ]`

**Review by wesm — COMMENTED** [review]
Not sure why the build failed, perhaps transient

**Comment by jakirkham** [comment]
The feedstock needs to be [re-rendered]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ).

**Comment by wesm** [comment]
+1

**Comment by jakirkham** [comment]
Feel free to add yourselves as maintainers if it is of interest.

File: recipe/build.sh [hunk 2]
```diff
@@ -9,8 +9,16 @@
 #
 source activate "${CONDA_DEFAULT_ENV}"
```
Comment (jakirkham) [inline recipe/build.sh:10]: This top section can be cut. It happens automatically in `conda-build` now.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -7,10 +7,10 @@ package:
 source:
   url: https://github.com/google/snappy/releases/download/{{ version }}/snappy-{{ version }}.tar.gz
   fn: snappy-{{ version }}.tar.gz
-  md5: 7358c82f133dc77798e4c2062a749b73
+  sha256: 134bfe122fd25599bb807bb8130e7ba6d9bdb851e0b16efcb83ac4f5d0b70057
 
 build:
-  number: 2
+  number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml:13]: This should go to `0` since the version number was bumped.

File: recipe/build.sh [hunk 4]
```diff
@@ -9,8 +9,16 @@
...
 
+autoreconf --force --install
 ./autogen.sh
-./configure --with-pic --prefix=$PREFIX
+if [ "$(uname)" == "Linux" ]; then
+  export CXXFLAGS="${CXXFLAGS} -O2 -DNDEBUG"
+else
+  # There is at least on OSX a bug affecting -O2 builds
+  export CXXFLAGS="${CXXFLAGS} -O1 -DNDEBUG"
```
Comment (jakirkham) [inline recipe/build.sh:18]: Should we raise this upstream? Or is there already an issue we can follow?

Comment (xhochy) [reply recipe/build.sh:18]: Already raised upstream, as they don't seem to have a bug tracker, posted to the ML: https://groups.google.com/forum/#!topic/snappy-compression/TD-x9gt6mjw Sadly no response yet.

**Comment by xhochy** [comment]
@wesm I would add myself as a maintainer, should I add you too?

**Comment by wesm** [comment]
Yes please. No need to wait for the build to run again on OS X so maybe put [ci skip]

File: recipe/build.sh (hunk 4 above)
Comment (jakirkham) [reply recipe/build.sh:18]: Thanks for doing that.

**Comment by jakirkham** [comment]
No worries. I'll do it in a subsequent PR. Just please confirm that it is ok once it is open.

**Comment by jakirkham** [comment]
Please see PR ( https://github.com/conda-forge/snappy-feedstock/pull/8 ).

**Pull request merged**